README.md

```diff
@@ -1,27 +1,87 @@
+| `network_management_nameservers`          | list of strings              | `['8.8.8.8', '8.8.4.4']` | List of all nameservers to use                                                                                                                     |     N    |
+| `network_management_clear_bridges`        | boolean                      | `False`                  | Remove all ovs bridges before recreating them. This is useful for renaming bridges.                                                                |     N    |
+| `network_management_reboot_for_config`    | boolean                      | `False`                  | Reboot target server to setup new network config, useful for major network configuration, which may require manual interactions otherwise          |     N    |
+| `interfaces`                              | [list of dicts](#interfaces) | `[]`                     | List of all interfaces to setup, keep in mind it can cause various errors if you configure a interface here and later use it as a port on a bridge |     N    |
+| `bridges`                                 | [list of dicts](#bridges)    | `[]`                     | List of network bridges to setup (all bridges are managed by openvswitch)                                                                          |     N    |
+
+### type
+Defines how a network or bridge should be configured. Possible values are:
+
+| Value    | Description                            | Side effects                                  |
+|----------|----------------------------------------|-----------------------------------------------|
+| `manual` | Do nothing but bring up interface link | Ignore options `ips`, `default_gw` and `dhcp` |
+| `static` | Configure interface statically         | Ignore option `dhcp`                          |
+| `dhcp`   | Configure interface via dhcp           | Ignore option `ips` and `default_gw`          |
+
+
+### dhcp
+| Option      | Type    | Default                  | Description                                                                        | Required |
+|-------------|---------|--------------------------|------------------------------------------------------------------------------------|:--------:|
+| `hostname`  | string  | `{{ ansible_hostname }}` | Hostname to be requested (is ignored in `network_management_default_dhcp_options`) |     N    |
+| `leasetime` | integer |                          | Request a specific lease time in seconds.                                          |     N    |
+| `metric`    | integer |                          | Metrics are used to prefer an interface over another one, lowest wins.             |     N    |
+
+
+### port
+| Option | Type   | Default | Description                     | Required |
+|--------|--------|---------|---------------------------------|:--------:|
+| port   | string |         | Interface name to add to bridge |     Y    |
```

Comment:
please add the `broadcast` key you are using.

Reply:
you mean the `vlan` option

Reply:
No I meant the `broadcast` key but commented at the wrong position. It has to be in the `interfaces` section.

Reply:
still